### DISCUSSION TITLE
ethernet config?

### DISCUSSION THREAD

**Discussion by kenrector** [ldijkman/WT32-ETH01-LAN-8720-RJ45-#14] [Q&A]
I use mac OS X Catalina.  I have a WT32-ETH01 connected to a simple ft232 type USB to serial board as shown here and at other sites.  After connection I see esp logging output on the dev/tty.usb* device when I connect and disconnect the rj45 cable.
So far, so good.

  I use espidf tools to flash the esp with the ESP32 HelloWorld example.  So far, so good.

I would like to run the ESP32 basic ethernet example.

 Can anyone help me with configuration settings?

**Comment by ldijkman** [comment]
Hello

at the moment all my free time goes into Electra
https://github.com/ldijkman/randomnerd_esp32_wifi_manager

maybe later i will try to get Electra running on the wired esp
